### PR TITLE
Feature/update/dependencies

### PR DESCRIPTION
**Pull request by janwidmer**
## Pull request
Updates dependencies around stylelint and eslint
 
### Ticket
-
 
### Browser testing
-
 
### Checklist
- [ ] I merged the current development branch (before testing)
- [ ] Added JSDoc and styleguide demo
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Did run automated tests and linters
- [ ] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [x] Did review code and documentation
- [ ] Tested all links in project relevant browsers
- [ ] Tested all links in different screen sizes
- [ ] Did check accessibility (Wave, only errors)
- [ ] Re-assign ticket to developer

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@ import 'core-js/stable';
 // NOTE: this is not working for styleguidist. There you need to add style imports to the required configuration
 import './setup/_scss.scss';
 
+// vendor styles
+import 'swiper/css';
```
Comment (patric-eberle) [inline src/main.ts:8]: Can't we import them inside of `./setup/_scss.scss`?

**Review by patric-eberle — CHANGES_REQUESTED** [review]
Good work. Only two small things. I also discovered, that some Devtools API is part of the build. Can this be removed?
grafik

**Comment by patric-eberle** [comment]
...and PHPStorm gives me this error when looking at a TS file.

grafik

**Comment by janwidmer** [comment]
> ...and PHPStorm gives me this error when looking at a TS file.
> 
> grafik

I know, I also had this one.. have researched a little bit without great outcome. My Assumption is, that the PhpStorm EsLint Plugin is not yet fully compatible with the new Version of EsLint, as the Problem only appears for the PhpStorm EsLint and not in the CLI.. As I don't have a solution, the only way I see would be to wait for next PhpStorm Versions.. What do you think?

File: src/main.ts (hunk 1 above)
Comment (janwidmer) [reply src/main.ts:8]: No, the import does not seem to work in scss: 
image
The changed the structure and Package Type of swiper, see here: https://swiperjs.com/migration-guide

**Pull request merged**